propagation/toolboxcloud.md

```diff
@@ -1,10 +1,25 @@
 
-The process begins with an assessment of your request to estimate the effort required to cloudify the toolbox. The main
-advantage of this approach is that the specific functionality will also become available as a modular building block for
-other projects. Therefore, it is crucial to provide a compelling justification for why this feature deserves to be 'cloudified'.
+Service users may select functionalities from a toolbox that they would like to use on the APEx platform. The proposed functionalities and their toolbox will then be reviewed by ESA and APEx experts. Similar to the algorithm enhancement service, an effort estimation and an offer will be made following the review. On acceptance of the offer, the requested functionalities will be provided as an Application Package according to OGC best practice, or as an openEO User Defined Function or Process (UDF/UDP). 
 
+To ensure interoperability between procedures from different toolboxes, it is necessary to share data using standard data formats, writing intermediary files to disk. Users should consider if the flexibility of this approach outweighs the performance drawbacks and request deployment as a single, integrated package instead, if performance is critical. 
+
+Functionalities made available through the toolbox cloudification service are hosted via the APEx algorithm hosting service. A similar set of features is offered by the algorithm enhancement service, which focuses on improving maturity and performance of algorithms and preparing them for hosting on APEx. The algorithm enhancement service works together with scientists and algorithm developers to make their algorithms available on APEx. 
+
+#### Features of the Toolbox Cloudification Service 
+
+The toolbox cloudification service offers an analysis of the implementation of the toolbox and the requested processing capability. On the basis of this analysis, an implementation strategy is proposed and executed to provide the functionality outside the context of the toolbox in a fully cloud-integrated format on the APEx platform. 
+
+Projects may benefit from the toolbox cloudification service by bringing key functions from traditional desktop EO toolboxes into the cloud, where they can be integrated into large-scale processing workflows. 
+
+Toolboxes evolve over time. If a toolbox migrated to APEx, it shall be kept up to date. This requires maintenance. 
+
+#### How to use the Toolbox Cloudification Service 
+
+In order to use the toolbox cloudification service, a proposal specifies a candidate toolbox and selects key functionality for cloudification. The proposal should contain a short justification explaining the need for the selected functionality. ESA reviews the proposal and submits selected proposals to the APEx team for analysis and price offer. 
```

Comment:
> a proposal specifies a candidate toolbox and selects key functionality for cloudification

Maybe it is good to explicitly mention that a proposal needs to be submitted to the APEx team. The actual contact information (how to submit the proposal) can be added later when we have a bit more information on this.